src/GlobalScreen/Client/src/GS.ts

```diff
@@ -0,0 +1,200 @@
+namespace il {
+
+    /**
+     * Module GlobalScreen
+     */
+    export module GS {
+
+
+        /**
+         * Logger
+         */
+        class Logger {
+            private static readonly debug = true;
+
+            static log(item: any): void {
```

Comment:
Maybe we should think about a global logger being provided in the `il` JavaScript-*namespace*. I also used my own logger here `./Services/OnScreenChat/js/notifications.js` (in `VanillaJS` :-)), but maybe we should create a central logging component (maybe in `TypeScript`, if we have a general JS/TS strategy) for future ILIAS versions.